<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Login Test"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="8.950">
  <properties />
            <testcase name="loginTest" time="8.950" classname="com.w2a.testcases.LoginTest">
                <failure type="java.lang.AssertionError"
                  message="Login test failed"
              >
<![CDATA[java.lang.AssertionError: Login test failed
  org.testng.Assert.fail(Assert.java:93)
  com.w2a.testcases.LoginTest.loginTest(LoginTest.java:20)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
